index.d.ts

```diff
@@ -0,0 +1,58 @@
+
+/**
+  This function is used to determine the filename for each downloaded file.
+  @param options - The options for renaming.
+  @returns The new filename for the downloaded file.
+*/
+export type RenameFunction = (options: {url: string; index: number; urls: string[]}) => string;
+
+/*
+  Options for the multiDownload function.
+ */
+export type MultiDownloadOptions = {
+	/**
+	  The function to rename the downloaded files.
+	  @default undefined (use original filenames)
+	 */
+	rename?: RenameFunction;
+	/**
+	  The delay time between each file download in milliseconds.
+	  @default 1000
+	 */
+	downloadInterval?: number;
+};
+
+/**
+  Download multiple files from an array of URLs.
+  @param urls - The URLs to download.
+  @param options - The options for downloading.
+  @returns A promise that resolves when all files have been downloaded.
+  @example
+  ```
+  import { multiDownload } from 'multi-download';
+
+  async function downloadFiles() {
+    const urls = [
```

Comment:
Use tab-indentation for the code example.

Reply:
seems already tab indend